crates/evm/traces/src/inspector.rs

```diff
@@ -128,32 +123,25 @@ impl Tracer {
         step.gas_cost = step.gas - interp.gas.remaining();
 
         // Error codes only
-        if status as u8 > InstructionResult::OutOfGas as u8 {
-            step.error = Some(format!("{status:?}"));
+        if interp.instruction_result as u8 > InstructionResult::OutOfGas as u8 {
```

Comment:
Can't we somehow wrap this into some way to check this for an err? Ideally on revm, as this seems error prone